### PR TITLE
The price is changing

### PR DESCRIPTION
**Pull request by darknight2590**
Adding a new economic variable cost
The price is changing

### CODE REVIEW

**Comment by darknight2590** [comment]
A new idea for more realism
Add a variable price per liter of gasoline
To add more realism

File: source/fuel_client.lua [hunk 1]
```diff
@@ -2,7 +2,8 @@ local isNearPump = false
 local isFueling = false
 local currentFuel = 0.0
 local currentCost = 0.0
-local currentCash = 1000
+local todaycoast = 0
```
Comment (TechnoBehemoth) [inline source/fuel_client.lua]:
```suggestion
local todaycost = 0
```
Typo fix for clarity

File: source/fuel_client.lua [hunk 2]
```diff
@@ -108,8 +109,9 @@ AddEventHandler('fuel:startFuelUpTick', function(pumpObject, ped, vehicle)
 			isFueling = false
 		end
 
-		currentCost = currentCost + extraCost
-
+		if extraCost >= 1 then
+			currentCost = currentCost + extraCost
+			todaycoast = extraCost
```
Comment (TechnoBehemoth) [inline source/fuel_client.lua]:
```suggestion
			todaycost = extraCost
```

File: source/fuel_client.lua [hunk 3]
```diff
@@ -142,10 +144,11 @@ AddEventHandler('fuel:refuelFromPump', function(pumpObject, ped, vehicle)
 
 		if pumpObject then
 			local stringCoords = GetEntityCoords(pumpObject)
-			local extraString = "\n" .. Config.Strings.TotalCost .. ": ~g~$" .. Round(currentCost, 1)
+			local extraString = "\n" .. Config.Strings.TotalCost .. ": ~g~$" .. Round(todaycoast, 1)
```
Comment (TechnoBehemoth) [inline source/fuel_client.lua]:
```suggestion
			local extraString = "\n" .. Config.Strings.TotalCost .. ": ~g~$" .. Round(todaycost, 1)
```

**Review by eralpozcan — CHANGES_REQUESTED** [review]
Please make the changes.

**Pull request merged**